tests/nested/manual/muinstaller-real/task.yaml

```diff
@@ -5,8 +5,9 @@ details: End-to-end test for install via muinstaller
 systems: [ubuntu-22.04-64]
 
 environment:
-  # No partial gadget by default
+  # No partial gadget or TPM by default
   PARTIAL_GADGET: false
+  NESTED_ENABLE_TPM: false
```

Comment:
In general I think we should prefer to test things with TPM except one ("plain") case